src/System.Console/src/System/Console.cs

```diff
@@ -541,37 +542,37 @@ public static void WriteLine(ulong value)
         }
 
         [MethodImplAttribute(MethodImplOptions.NoInlining)]
-        public static void WriteLine(object value)
+        public static void WriteLine(object? value)
         {
             Out.WriteLine(value);
         }
 
         [MethodImplAttribute(MethodImplOptions.NoInlining)]
-        public static void WriteLine(string value)
+        public static void WriteLine(string? value)
         {
             Out.WriteLine(value);
         }
 
         [MethodImplAttribute(MethodImplOptions.NoInlining)]
-        public static void WriteLine(string format, object arg0)
+        public static void WriteLine(string format, object? arg0)
         {
             Out.WriteLine(format, arg0);
         }
 
         [MethodImplAttribute(MethodImplOptions.NoInlining)]
-        public static void WriteLine(string format, object arg0, object arg1)
+        public static void WriteLine(string format, object? arg0, object? arg1)
         {
             Out.WriteLine(format, arg0, arg1);
         }
 
         [MethodImplAttribute(MethodImplOptions.NoInlining)]
-        public static void WriteLine(string format, object arg0, object arg1, object arg2)
+        public static void WriteLine(string format, object? arg0, object? arg1, object? arg2)
         {
             Out.WriteLine(format, arg0, arg1, arg2);
         }
 
         [MethodImplAttribute(MethodImplOptions.NoInlining)]
-        public static void WriteLine(string format, params object[] arg)
+        public static void WriteLine(string format, params object?[]? arg)
```

Comment:
I think we agreed that params arrays should be not nullable as the compiler will transform it to an empty array, @stephentoub am I remembering correctly?

Reply:
I'm ok either way here, but as the code goes out of its way to make null work, I think we should stick with the `object?[]?` annotation used here.  Many places where params is used, we explicitly throw an exception if null is passed, which is probably what @safern is thinking of.